dicodile/workers/dicod_worker.py

```diff
@@ -112,7 +113,7 @@ def compute_z_hat(self):
                 k0, pt0, dz = None, None, 0
 
             # update the accumulator for 'random' strategy
-            accumulator = max(abs(dz), accumulator)
+            accumulator += abs(dz)
```

Comment:
Why do you change this? It should not be a sum here but the max over all segments